AsyncImageLoader.Avalonia.Demo/Pages/AdvancedImagePage.axaml

```diff
@@ -8,15 +8,17 @@
              x:Class="AsyncImageLoader.Avalonia.Demo.Pages.AdvancedImagePage">
   <Grid ColumnDefinitions="* 8 150 150" RowDefinitions="Auto Auto Auto Auto"
         HorizontalAlignment="Center">
-    <TextBlock Grid.Column="2" Grid.Row="0" HorizontalAlignment="Center">AdvancedImage</TextBlock>
-    <TextBlock Grid.Column="3" Grid.Row="0" HorizontalAlignment="Center">Image</TextBlock>
+    <TextBlock Grid.Column="2" Grid.Row="0" HorizontalAlignment="Center" Text="AdvancedImage"/>
+    <TextBlock Grid.Column="3" Grid.Row="0" HorizontalAlignment="Center" Text="Image"/>
```

Comment:
I thing what moving controls content is an unnecessary change, not related to compatibility with avalonia 11

Reply:
In the latest Avalonia 0.11 version. They are not going to support putting text like

```xaml
<TextBlock>xxx</TextBlock> # This will throw the error
```

As a result, we have to put all the texts in the Text property.